finder/finder.cc

```diff
@@ -42,18 +32,25 @@ Status FinderServiceImpl::CheckFood(ServerContext* context,
                                     ShopResponse* response) {
   std::cout << "========== Receving Request Food: " << request->food_name()
             << " ==========" << std::endl;
-
-  vector<ShopInfo> result = ProcessRequest(request->food_name());
+  const opencensus::trace::Span& span = grpc::GetSpanFromServerContext(context);
+  span.AddAnnotation("Processing request.");
+  absl::SleepFor(absl::Milliseconds(10));
```

Comment:
It for demo purpose: adding latency for each stage of the span so that the trace is more interpretable. A request only takes 1~2ms without `SleepFor`, and the visualized span in Cloud Trace is too short.